pkg/aws_client/aws_v2/instance.go

```diff
@@ -43,7 +43,8 @@ func (client *AwsV2Client) LaunchInstance(subnetID string, imageID string, count
 
 // ListInstance pass parameter like
 // map[string][]string{"vpc-id":[]string{"<id>" }}, map[string][]string{"tag:Name":[]string{"<value>" }}
-func (client *AwsV2Client) ListInstance(filters ...map[string][]string) ([]types.Instance, error) {
+// instanceIDs can be empty. And if you would like to get more info from the instances like security groups, it should be set
+func (client *AwsV2Client) ListInstance(instanceIDs []string, filters ...map[string][]string) ([]types.Instance, error) {
```

Comment:
could you rename it to `ListInstances` as we expect returning a slice of instances ?